web/src/main/java/com/softserve/osbb/controller/RoleController.java

```diff
@@ -115,15 +119,19 @@
         return new ResponseEntity<>(HttpStatus.OK);
     }
     private Resource<Role> addResourceLinkToRole(Role role) {
-        if (role == null) return null;
+        if (role == null) {
+            return null;
```

Comment:
just wondering, is it valit to return null here (and few more places in this PR) ?